Nbuild/Program.cs

```diff
@@ -99,6 +107,78 @@ static int Main(string[] args)
         return (int)result.Code;
     }
 
+    /// <summary>
+    /// Run the build targets based on the provided arguments.  This method is added for convenience so that 
```

Comment:
The RunBuildTargets method is  new method to make code maintainable.  Code was embeded in Main method